### PR TITLE
Update env vars for all components

### PR DESCRIPTION
**Pull request by adrian-fitsios**
[AB#7141](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7141)
[AB#6529](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6529)
[AB#7053](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7053)

### CODE REVIEW

File: Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml [hunk 1]
```diff
@@ -32,17 +32,20 @@ spec:
...
           ##Public browse base URL
         - name: PUBLIC_BROWSE_BASE_URL
           value: "https://buyingcataloguedev.uksouth.cloudapp.azure.com"
 
           #Ordering API base URL
         - name: ORDER_API_URL 
           value: "http://orapi:5104"
+
+          ## URL for Azure Storage Account
+        - name: BLOBSTORE_HOST
```
Comment (stuart-bass-cgi) [inline Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml:44]: order form has this config... but doesn't need it. Can you please remove?

File: local-helm/buyingcatalogue/charts/isapi/templates/deployment.yaml [hunk 2]
```diff
@@ -115,6 +115,8 @@ spec:
                   key: {{ .hostUriConfig.key | quote }}
             - name: SMTPSERVER__PORT
               value: {{ .port | quote }}
+            - name: SMTPSERVER__EMAILSUBJECTPREFIX
```
Comment (stuart-bass-cgi) [inline local-helm/buyingcatalogue/charts/isapi/templates/deployment.yaml:118]: Can you please copy these over to platform-helm?

**Review by stuart-bass-bjss — DISMISSED** [review]

File: Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml (hunk 1 above)
Comment (stuart-bass-cgi) [reply Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml:44]: actually, I'm wrong.

**Review by stuart-bass-bjss — APPROVED** [review]

File: local-helm/buyingcatalogue/charts/isapi/templates/deployment.yaml (hunk 2 above)
Comment (adrian-fitsios) [reply local-helm/buyingcatalogue/charts/isapi/templates/deployment.yaml:118]: can & will do 😄

**Pull request merged**